pkg/bff/gds_test.go

```diff
@@ -62,70 +65,173 @@ func (s *bffTestSuite) TestLookup() {
 	require.NotEmpty(rep.TestNet, "expected testnet result from server")
 }
 
-func (s *bffTestSuite) TestRegister() {
+func (s *bffTestSuite) TestLoadRegisterForm() {
 	require := s.Require()
 
+	// Create initial claims fixture
+	claims := &authtest.Claims{
+		Email:       "leopold.wentzel@gmail.com",
+		Permissions: []string{"read:nothing"},
+	}
+
+	// Endpoint must be authenticated
+	_, err := s.client.LoadRegistrationForm(context.TODO())
+	require.EqualError(err, "[401] this endpoint requires authentication", "expected error when user is not authenticated")
```

Comment:
Would be nice if we could separate the status check from the error message check, or have a helper to construct and check the expected response, but I think this is fine for right now.

Reply:
I agree - though @DanielSollis did make a good case that testing the string is technically testing both since the status is in the string. Still, it feels more explicit to have an int check and then a message check because then the message check can be optional (e.g. if it's a custom message based on the input).